Drivers/Src/CAN.c

```diff
@@ -0,0 +1,120 @@
+#include "CAN.h"
+
+static void floatToBytes(float val, uint8_t* bytes_array, uint8_t bytes);
+
+/**
+ * @brief   Initializes the CAN system
+ * @param   None
+ * @return  None
+ */
+
+// Not sure if we want CAN1 to be an argument to CAN_Init or not

```

Comment:
This initialization should always initialize CAN1, so it's good as it is right now